reference_simulations/get_gleam.py

```diff
@@ -6,6 +6,9 @@
 import os
 import sys
 
+print("This script is deprecated and will be removed in the future.")
+print("Please used scripts/download_gleam.py in pyradiosky instead.")
```

Comment:
Please use the `warnings` module to issue a `DeprecationWarning` instead of printing.

Reply:
Done.